### DISCUSSION TITLE
How to transfer information from on cluster assignment from one dataset to another?

### DISCUSSION THREAD

**Discussion by DRG0209** [satijalab/seurat#4580] [Q&A] [answered]
Hi, 
I'm a wet lab researcher trying to get more into bioinformatics, but I'm still a beginner - so please excuse if this question sounds trivial to some of you! :)

We used two different single cell methods (10X and SMARTseq) on the same tissue and first of all, I analysed both datasets separately. Then I used Harmony to integrate them. Now I would like to label cells based on where they clustered in the separate analyses. For instance, I would like to see in which cluster cells from cluster 8 in the SMARTseq- or cluster 10 in the 10X-dataset end up when both are integrated.

I know that I can extract cell IDs using "WhichCells" and that I can add information to a Seurat object using "AddMetaData". Do I have to combine these two in some way? 

Many thanks in advance! 

Best,
David

**Comment by timoast** [comment, marked as answer]
Please see the label transfer vignettes on the Seurat website: https://satijalab.org/seurat/articles/integration_mapping.html
